tests/test_routes.py

```diff
@@ -276,3 +306,21 @@ def test_add_item_no_order(self):
             content_type="application/json",
         )
         self.assertEqual(resp.status_code, status.HTTP_404_NOT_FOUND)
+
+    def test_get_item_item_not_found(self):
+        """
+        When there is an order but no such item.
+        It should not Read an item thats not found
+        """
+        order = self._create_orders(1)[0]
+        print(order.id)
+        resp = self.app.get(f"{BASE_URL}/{order.id}/items/0")
+        print(resp.status_code)
+        self.assertEqual(resp.status_code, status.HTTP_404_NOT_FOUND)
+    
+    def test_get_item_order_not_found(self):
+        """
+        It should not Read an item when the order is not found
+        """
+        resp = self.app.get(f"{BASE_URL}/0")
```

Comment:
Instead of using the read order URL, you should use the same read item URL. The purpose of this test is to verify that the API returns an error response if the order is not present in the database.